tests/PhpGeneratorBundle/RestClient/Fixtures/raml/returns-money/api.raml

```diff
@@ -0,0 +1,17 @@
+#%RAML 1.0
+title: Accounts
+version: 1.0
+baseUri: https://example.com/accounts/rest/v1
+mediaType: application/json
+protocols: [ HTTPS ]
+
+uses:
+  Paysera: https://raw.githubusercontent.com/paysera/lib-raml-common/0.0.5/rest.raml
```

Comment:
Please do not use remote urls in tests. Take a look at `tests/PhpGeneratorBundle/RestClient/Fixtures/raml/account/api.raml` - use local copy of this library.